### DISCUSSION TITLE
Is there a plan yet for FS22?

### DISCUSSION THREAD

**Discussion by TSO-Original** [Stephan-S/FS19_AutoDrive#2128] [Ideas]
Not wanting to start FS22 yet, waiting for bugs to come out and hoping to see autodrive for fs22. Is there any thoughts about whether you will port it over?

**Comment by Yllelder** [comment]
I guess until they release the Giants Editor, there is little that can be done.

I'm looking forward to it too.

**Reply by Roby1164** [reply]
Giants Editor 9 was released overnight.
Those of the Giants, they tried to copy it, but they did a bad job, it also applies to Course Play.
We are waiting calmly for the FS22 version of both.

**Reply by Nomat50** [reply]
Hello everyone, here is a link to GIANTS_Editor_9.0.1 https://modsfire.com/download/5534o08SY5T0nLn/20bd1

**Reply by nuruevural** [reply]
Привет мастерам ! Пари когда планируете авто драйв наладить ?
